app/src/main/java/com/jerboa/model/SiteViewModel.kt

```diff
@@ -228,7 +228,8 @@ class SiteViewModel(private val accountRepository: AccountRepository) : ViewMode
     fun voteDisplayMode(): LocalUserVoteDisplayMode {
         return when (val res = siteRes) {
             is ApiState.Success ->
-                res.data.my_user?.local_user_view?.local_user_vote_display_mode ?: LocalUserVoteDisplayMode.default()
+                res.data.my_user?.local_user_view?.local_user_vote_display_mode
+                    ?: LocalUserVoteDisplayMode.default(res.data.my_user?.local_user_view?.local_user?.show_scores)
```

Comment:
This works. Eventually that field will go away.

Reply:
0.20 In Lemmy, yes. But It will always exist in 0.19. And always in some form in the LemmyApi datatypes. So it can be mapped to from older versions. I'll probably have a legacy namespace for that.